### DISCUSSION TITLE
Allowing uper case table names

### DISCUSSION THREAD

**Discussion by bdelathouwer** [sqlalchemy/sqlalchemy#8338] [Usage Questions]
Dear, 

I have a question regarding the use of upper case table names.
Is there a way to allow the use of upper case table names in SQL alchemy?
Because now my table names get automaticly converted to lower case ones while this is not what I want.
 I'm using sqlalchemy 1.4.39 and an sql version 5.7.36
Thanks in advance and with best regards,
Ben

**Comment by CaselIT** [comment]
Hi,

Following the table documentation does not work? https://docs.sqlalchemy.org/en/14/core/metadata.html?highlight=table#sqlalchemy.schema.Table.params.name

> I'm using sqlalchemy 1.4.39 and an sql version 5.7.36

Not sure what kind of db "sql version 5.7.36" is

**Comment by bdelathouwer** [comment]
Dear, 

Thank you for your sugestion!
I will try it asap and report back to you!

Kind regards,
Ben
